src/__mocks__/exchange.ts

```diff
@@ -1,19 +1,21 @@
-import { getBalances } from "../exchange"
+// TODO: Fix this test. It's using a non-existing import
 
-it('fetchBalance', async () => {
-    const balance = expect.objectContaining({
-        USD: expect.objectContaining({
-            total: expect.any(Number), // how to set toBeGreaterThanOrEqual(0) ?
-            exchange: expect.any(Number),
-        }),
-        BTC: expect.objectContaining({
-            total: expect.any(Number), // how to set toBeGreaterThanOrEqual(0) ?
-            exchange: expect.any(Number),
-            onchain: expect.any(Number),
-            offchain: expect.any(Number),
-        }),
-    })
+// import { getBalances } from "../exchange"
```

Comment:
I'm wondering why this test even passed. I think it can be deleted I'm not sure what it was doing but clearly outdated and no longer used.

Reply:
I think it was not run at all.